__tests__/integration/one-app.spec.js

```diff
@@ -799,30 +799,29 @@ describe('Tests that can run against either local Docker setup or remote One App
           expect(url).toMatch(/healthy-frank\/simple/);
         });
 
-        test('loadModuleData is not called when navigating to module on client', async () => {
+        test('data loads when navigating to module on client', async () => {
           // start by navigating to ssr-frank without prefetch
           await browser.url(`${appInstanceUrls.browserUrl}/healthy-frank`);
-          const noPrefetchLink = await browser.$('.ssr-frank-link');
-          await noPrefetchLink.click();
+          const prefetchLink = await browser.$('.prefetch-ssr-frank');
```

Comment:
Ideally this should be testing navigating without potential prefetching.

Reply:
Fixed in f1b0889 by modifying test to be more specific.